test/converter.test.js

```diff
@@ -19,9 +19,12 @@ test('it should return english spelling if no language specified', t => {
 });
 
 test('its should throw error if provided language not supported', t => {
-	t.throws(() => {
-		const converter_xx = converter('xx');
-		converter_xx(200);
-	}, null,
-	'Cannot find converter for language: xd');
+	t.throws(
+		() => {
+			const converter_xx = converter('xx');
+			converter_xx(200);
+		},
+		null,
+		'Cannot find converter for language: xd',
```

Comment:
`xd` or `xx`?

Reply:
Apparently this test doesn't really check the error message, I've changed it so it asserts the message in a different way